plugins/common/opcua/client.go

```diff
@@ -18,6 +18,10 @@ type OpcUAWorkarounds struct {
 	AdditionalValidStatusCodes []string `toml:"additional_valid_status_codes"`
 }
 
+type OpcUAAdditionalFields struct {
+	IncludeDataType bool `toml:"include_datatype"`
+}
```

Comment:
Not sure if this needs to be a new sub-struct.. (Just a comment, I'm open to other opinions)

Reply:
My thought here is that OPC-UA nodes have multiple attributes that can be read by a client including DataType, Description, namespace information, Quality (arguably). Putting them in a separate structure adds some organization as I anticipate future development will implement reading more of these attributes.

Reply:
Changed to a list